cluster-provision/k8s/1.19/provision.sh

```diff
@@ -59,54 +69,33 @@ yum -y install iscsi-initiator-utils
 # To prevent preflight issue realted to tc not found
 dnf install -y tc
 
-# Install docker required packages.
-dnf -y install yum-utils \
-    device-mapper-persistent-data \
-    lvm2
-
-# Add Docker repository.
-dnf config-manager --add-repo=https://download.docker.com/linux/centos/docker-ce.repo
+export CRIO_OS=CentOS_8_Stream
+export CRIO_VERSION=1.19
+curl -L -o /etc/yum.repos.d/devel:kubic:libcontainers:stable.repo https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/$CRIO_OS/devel:kubic:libcontainers:stable.repo
+curl -L -o /etc/yum.repos.d/devel:kubic:libcontainers:stable:cri-o:$CRIO_VERSION.repo https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable:cri-o:$CRIO_VERSION/$CRIO_OS/devel:kubic:libcontainers:stable:cri-o:$CRIO_VERSION.repo
+dnf install -y cri-o
 
-# Enable the nightly version to get a new enough runc version which includes containerd
-# with a new enough runc version to avoid https://github.com/kubernetes/kubernetes/issues/95296#issuecomment-714419461
-# which manifests in /dev/* permission denied flakes on cpu manager enabled nodes
-dnf config-manager --set-enabled docker-ce-nightly
+# Install crictl
+(
+cd /usr/bin/
+curl -L https://github.com/kubernetes-sigs/cri-tools/releases/download/v${CRIO_VERSION}.0/crictl-v${CRIO_VERSION}.0-linux-amd64.tar.gz | tar xvz
+)
 
-# Install package(s) that trigger the enablement of the container-tools yum module
-dnf -y install container-selinux
+# "crio pull" and "docker pull" is needed in test repos to pre-pull images
```

Comment:
this is tricky, are there other ways instead ? 
can you please add a link for one example ?

Reply:
I'm basing this on a draft PR authored by Roman https://github.com/kubevirt/kubevirtci/pull/467/files#diff-62b367f592f4e1340e2ebcd641372ffd5f114cda29d2783025798d35848daaf1R76
So I'm not actually sure, @rmohr  could you please clarify?

Reply:
> this is tricky, are there other ways instead ?

What is tricky? The two tools are almost 100% identical on the commandline. For instance kubevirt/kubevirt allows you to run the containerized builds with docker and crio this way.

> So I'm not actually sure, @rmohr could you please clarify?

This is needed to allow a smooth transition. People use in their repos `docker pull` already in various places. This allows a smoother transition. I am not sure if it is worth the time to invest into more complex mechanisms.

Reply:
Maybe we could come up with some adapter to interact with diffent container engine clients

Reply:
> Maybe we could come up with some adapter to interact with diffent container engine clients

How would that make things simpler?

Reply:
> How would that make things simpler?

No it wouldn't, it would just give a way to deal with possible differences between the cli commands.
But as of now, it probably isn't neccessary as both tools are almost the same